__tests__/app.test.js

```diff
@@ -39,13 +29,82 @@ describe("GET /api/topics ", () => {
     });
 });
 
-describe("GET /api/topic ", () => {
+describe("GET /api/topicz ", () => {
   test("when path error returns a 404 for bad path", () => {
     return request(app)
-      .get("/api/topic ")
+      .get("/api/topicz ")
       .expect(404)
       .then(({ body }) => {
         expect(body.msg).toBe("bad request sorry");
       });
   });
-});
\ No newline at end of file
+});
+
+describe('GET /api/articles', () => { 
+  test(' should return status 200 and an array of article objects with the comment_count,', () => { 
+    return request(app)
+      .get('/api/articles')
+      .expect (200)
+      .then(({ body }) => {
+         const { articles } = body;
+        expect(articles).toHaveLength(12);
+        expect(articles).toBeInstanceOf(Array);
+        articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              article_id: expect.any(Number),
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(String)
+            })
+          );
+        }); 
+
+    })
+  })
+   test('status 200, responds with array of article objects where it has a comment_count property', () => { 
+    return request(app)
+    .get('/api/articles')
+    .expect(200)
+      .then(({ body }) => {
+        const { articles } = body 
+        expect(articles[0]).toHaveProperty('comment_count')
+        expect(articles[0].comment_count).toEqual("2")
+
+      
+    })
+   })
+})
+describe('GET /api/articles', () => {
+  test('respondes with articles in order of date desc : ', () => {
```

Comment:
This test could be in the previous describe block as they both describe the same endpoint.